packages/components/src/components/Search/Search.tsx

```diff
@@ -145,7 +145,7 @@ export const Search = ({
                         </View>
                     }
                     rightAdornments={
-                        text ? (
```

Comment:
Isn't it the other way around? 
`text && !disabled` ?
Now it seems to only render the clear button if the search contains text AND is disabled